extensions/vscode/lsp/client/package.json

```diff
@@ -0,0 +1,23 @@
+{
+	"name": "lsp-wake-client",
+	"description": "VSCode part of a language server for wake",
+	"author": "SiFive",
+	"license": "MIT",
```

Comment:
wake is licensed under the Apache 2.0 license. The extension should presumably be the same.

Reply:
I changed it to Apache 2.0.